python/fleetx/utils/submitter.py

```diff
@@ -23,17 +39,32 @@ def __init__(self):
                           "--job-name {} " \
                           "--start-cmd 'sh start_job.sh' " \
                           "--job-conf config.ini " \
-                          "--files start_job.sh " \
+                          "--files start_job.sh {} " \
                           "--k8s-trainers {} {} " \
                           "--k8s-cpu-cores 35"
 
     def get_start_job(self, yml_cfg):
+        # set up config.ini
+        fs_name = yml_cfg['fs_name']
+        fs_ugi = yml_cfg['fs_ugi']
+        output_path = yml_cfg['output_path']
+        storage_type = yml_cfg['storage_type']
+        config = "storage_type = \"{}\"\n".format(storage_type)
+        config += "fs_name = \"{}\"\n".format(fs_name)
+        config += "fs_ugi = \"{}\"\n".format(fs_ugi)
+        config += "output_path = \"{}\"\n".format(output_path)
+        config += "FLAGS_rpc_deadline=3000000\n"
+        config += "NCCL_DEBUG=INFO\n"
+        with open("config.ini", "w") as fout:
+            fout.write(config)
+        os.system("cat config.ini")
```

Comment:
if you want to print config.ini for debug, just tell users where is the config.ini.